text_generation/causal_lm/cpp/continuous_batching/library/src/sampler.hpp

```diff
@@ -400,13 +270,11 @@ SamplerOutput Sampler::sample(std::vector<SequenceGroup::Ptr> & sequence_groups,
         ov::Tensor sequence_group_logits(ov::element::f32, ov::Shape{num_running_sequences, actual_seq_len, vocab_size}, (void *)sequence_group_logits_data);
 
         if (sequence_group->requires_sampling()) {
+            LogitProcessor logit_processor(sampling_params, sequence_group->get_unique_generated_ids(), sequence_group->get_unique_prompt_ids());
```

Comment:
BTW, maybe we can have a map of LogitsProcessors sequence_id -> LogitProcessor? 
In this case we can remove storing `get_unique_generated_ids` and `get_unique_prompt_ids` from sequence group, because this information is required only for sampler. 

It will be aligned with beam search case, where current sampling info is stored in sampler itself